cli/spark/endpoints/cluster_create.py

```diff
@@ -53,6 +53,11 @@ def execute(args: typing.NamedTuple):
 
     print_cluster_conf(cluster_conf)
 
+    spinner = utils.Spinner()
+
+    log.info("Please wait while your cluster is being provisioned")
+    spinner.start()
```

Comment:
you probably want to put this into a catch / finally block since any crash in the middle should be handled and the stop would be called as expected.